orchestra/migrations/0087_todo_task_data_migration.py

```diff
@@ -0,0 +1,28 @@
+# Generated by Django 2.2.13 on 2020-09-04 14:49
+
+from django.db import migrations
+from django.db.models import OuterRef
+from django.db.models import Subquery
+
+
+def fill_project_and_step_fields(apps, schema_editor):
+    Step = apps.get_model('orchestra', 'Step')
+    Project = apps.get_model('orchestra', 'Project')
+    Todo = apps.get_model('orchestra', 'Todo')
+    o = OuterRef('id')
```

Comment:
you'll have to explain this to me live :)

Reply:
specifically, why we need the [:1].

Perhaps a broader comment explaining what this code does will help.